microlens-pro/microlens-pro.cabal

```diff
@@ -0,0 +1,70 @@
+name:                   microlens-pro
+version:                0.1.0
+synopsis:               Prisms and isomorphisms for microlens
+description:
+  This package provides lens-compatible 'Prism' and 'Iso'. Consequently, it
+  depends on the rather heavy @profunctors@.
+  .
+  Thank you to the <https://hackage.haskell.org/package/lens lens> contributors
+  for the original code and some docs,
+  <https://cohomolo.gy Emily Pillmore> and
+  <https://mroman42.github.io/cosmoi Mario Román> for
+  <https://golem.ph.utexas.edu/category/2020/01/profunctor_optics_the_categori.html this post>
+  which inspired documentation, and Wikibooks contributors for
+  <https://en.wikibooks.org/wiki/Haskell/Lenses_and_functional_references#Isos this article>.
+  .
+  This package is a part of the
+  <https://hackage.haskell.org/package/microlens microlens> family; see the
+  readme <https://github.com/stevenfontanell/microlens on Github>.
+license:                BSD3
+license-file:           LICENSE
+author:                 Edward Kmett, Artyom Kazak, crumbtoo <lomiskiam@gmail.com>
+maintainer:             Steven Fontanella <steven.fontanella@gmail.com>
```

Comment:
I updated the author, maintainer and github links, please let me know if you see any issues with it. I changed it so that you're an author and I'm the maintainer, since it lives in this repo.

Reply:
Looks good!